src/me/vedang/clj_fdb/subspace/subspace.clj

```diff
@@ -41,11 +44,12 @@
   suffix."
   ([^Subspace s]
    (.pack s))
-  ([^Subspace s t]
+  ([^Subspace s k]
    (cond
-     (instance? Tuple t) (.pack s ^Tuple t)
-     (vector? t) (.pack s ^Tuple (ftup/create t))
-     :else (.pack s ^Tuple (ftup/from t)))))
+     (instance? byte-array-class k) (.pack s k)
+     (instance? Tuple k) (.pack s ^Tuple k)
+     (vector? k) (.pack s ^Tuple (ftup/create k))
+     :else (.pack s ^Tuple (ftup/from k)))))
 
```

Comment:
These changes are mixing a completely different thing into this PR. They are not related to adding KeySelector Support

Reply:
Moving it to a different PR.